docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,43 @@
+---
+id: getting-started
+title: Getting started with Tartiflette
+sidebar_label: 1. Getting started
+---
+
+Welcome, either you are a GraphQL newbie or an advanced user, Tartiflette will help you to build a powerful GraphQL API.
+
+This tutorial will explain to you the main features of Tartiflette and some of its advanced features _(dynamic introspection, SDL Directives ...)_ 
+
+## What're we doing?
+
+Nothing is better than to code to understand a library. Let's go to develop your **Tartiflette recipes manager GraphQL API**.
+
+Here are the topics we will cover during this getting started.
+
+* Install the Tartiflette library _(and sub-dependencies)_.
+* Declare your schema thanks to the SDL _(Schema Definition Language)_.
+* Create a Tartiflette _(Server)_ instance.
+* Write Query, Mutation and Subscription resolvers.
+* Run a Query, Mutation and Subscription request on your **Engine**.
+* Declare and use custom directive _(Advanced)_.
+* Secure nodes with directives _(Advanced)_.
+* How to expose different introspection based on context (Dynamic introspection) _(Advanced)_.
+
+The tutorial's result is available on github, [tartiflette-aiohttp-tutorial](https://github.com/dailymotion/tartiflette-aiohttp-tutorial).
+
+## Prerequisites
+
+We assume that you're familiar with Python 3.6+, asyncio and more generally with asynchronous development. We do not plan to link this tutorial with other technologies like RDMS _(MySQL, Postgres etc ...)_, we will keep this tutorial as much as possible focus on the tartiflette use. 
+
+Moreover, we will use intensively the command line to manipulate file or launch commands.
```

Comment:
we will intensively use the command line

Reply:
👍